packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -142,6 +161,7 @@ export const TestProviders = ({
                                                     value={addToCartModal}
                                                 >
                                                     {children}
+                                                    <AddToCartModal />
```

Comment:
Our TestProviders were missing this modal. We had the context in there (line 160), but not the modal to render yet.